caveclient/tools/table_manager.py

```diff
@@ -78,10 +79,9 @@ def get_all_view_metadata(client):
 def is_list_like(x):
     if isinstance(x, str):
         return False
-    try:
-        len(x)
+    if hasattr(x, "__len__"):
```

Comment:
this should be the same functionality but happy to change it back and noqa it if anyone is concerned